pkg/operation/care/constraints.go

```diff
@@ -252,6 +253,10 @@ func (c *Constraint) CheckForProblematicWebhooks(ctx context.Context) (gardencor
 	}
 
 	for _, webhookConfig := range validatingWebhookConfigs {
+		if strings.Contains(webhookConfig.Annotations[resourcesv1alpha1.OriginAnnotation], c.shoot.SeedNamespace) {
```

Comment:
Should we use the label selector here to select based on `resources.gardener.cloud/managed-by` label which was introduced with https://github.com/gardener/gardener/pull/6393?